### PR TITLE
add tensorboardX for loss visualization

### PR DESCRIPTION
**Pull request by KuNyaa**

### CODE REVIEW

File: fastNLP/core/trainer.py [hunk 1]
```diff
@@ -163,6 +167,11 @@ def _train_step(self, data_iterator, network, **kwargs):
             loss = self.get_loss(prediction, batch_y)
             self.grad_backward(loss)
             self.update()
+            self._summary_writer.add_scalar("loss", loss.item(), global_step=step)
+
+            if not self._graph_summaried:
```
Comment (choosewhatulike) [inline fastNLP/core/trainer.py]: what if some people don't want to visualize the loss? Could we add a switch flag

Comment (FengZiYjun) [reply fastNLP/core/trainer.py]: This is a recommended setting. Users will see loss curve if and only if they open tensorboard. We just prepare for it.
